source/mir/algebraic.d

```diff
@@ -728,7 +728,7 @@ Compatible with BetterC mode.
 alias Nullable(T...) = Variant!(typeof(null), T);
 
 /// ditto
-Nullable!T nullable(T)(T t)
+Nullable!T nullable(T)(return T t)
 {
     import core.lifetime: forward;
     return Nullable!T(forward!t);
```

Comment:
Unrelated, I know, but `forward` doesn't really make sense here, for a non-`auto ref` param. It always boils down to a `move` for non-ref params.